build-tools/packages/build-cli/src/instructionalPromptWriter.ts

```diff
@@ -45,56 +47,31 @@ interface Section {
+    ["@fluidframework/protocol-definitions", "https://dev.azure.com/fluidframework/internal/_build?definitionId=67"]
+  ]);
 
-export enum ADOPipelineLinks {
-    CLIENT = "https://dev.azure.com/fluidframework/internal/_build?definitionId=12",
-    SERVER = "https://dev.azure.com/fluidframework/internal/_build?definitionId=30",
-    BUILDTOOLS = "https://dev.azure.com/fluidframework/internal/_build?definitionId=14",
-    AZURE = "https://dev.azure.com/fluidframework/internal/_build?definitionId=85",
-    APIMARKDOWNDOCUMENTER = "https://dev.azure.com/fluidframework/internal/_build?definitionId=97",
-    BENCHMARK = "https://dev.azure.com/fluidframework/internal/_build?definitionId=96",
-    TESTTOOLS = "https://dev.azure.com/fluidframework/internal/_build?definitionId=13",
-    TINYLICIOUS = "https://dev.azure.com/fluidframework/internal/_build?definitionId=22",
-    BUILDCOMMON = "https://dev.azure.com/fluidframework/internal/_build?definitionId=3",
-    ESLINTCONFLIGFLUID = "https://dev.azure.com/fluidframework/internal/_build?definitionId=7",
-    COMMONDEFINITIONS = "https://dev.azure.com/fluidframework/internal/_build?definitionId=8",
-    COMMONUTILS = "https://dev.azure.com/fluidframework/internal/_build?definitionId=10",
-    PROTOCOLDEFINITIONS = "https://dev.azure.com/fluidframework/internal/_build?definitionId=67",
-}
 
 /**
  *
  * Returns ADO pipeline link for the releaseGroup
  */
-export const mapADOLinks = (releaseGroup: string | undefined): string => {
-    const adoLink = releaseGroup === "client"
-            ? ADOPipelineLinks.CLIENT
-            : releaseGroup === "server"
-            ? ADOPipelineLinks.SERVER
-            : releaseGroup === "azure"
-            ? ADOPipelineLinks.AZURE
-            : releaseGroup === "build-tools"
-            ? ADOPipelineLinks.BUILDTOOLS
-            : releaseGroup === "@fluid-tools/api-markdown-documenter"
-            ? ADOPipelineLinks.APIMARKDOWNDOCUMENTER
-            : releaseGroup ===  "@fluid-tools/benchmark"
-            ? ADOPipelineLinks.BENCHMARK
-            : releaseGroup === "@fluidframework/test-tools"
-            ? ADOPipelineLinks.TESTTOOLS
-            : releaseGroup === "tinylicious"
-            ? ADOPipelineLinks.TINYLICIOUS
-            : releaseGroup === "@fluidframework/build-common"
-            ? ADOPipelineLinks.BUILDCOMMON
-            : releaseGroup === "@fluidframework/eslint-config-fluid"
-            ? ADOPipelineLinks.ESLINTCONFLIGFLUID
-            : releaseGroup === "@fluidframework/common-definitions"
-            ? ADOPipelineLinks.COMMONDEFINITIONS
-            : releaseGroup === "@fluidframework/common-utils"
-            ? ADOPipelineLinks.COMMONUTILS
-            : ADOPipelineLinks.PROTOCOLDEFINITIONS;
-    return adoLink;
+export const mapADOLinks = (releaseGroup: ReleaseGroup | ReleasePackage | undefined): string | undefined => {
+    return releaseGroup === undefined ? "" : ADOPipelineLinks.get(releaseGroup);
```

Comment:
```suggestion
    return ADOPipelineLinks.get(releaseGroup);
```

I think you can just return. If the releaseGroup is undefined, then the `.get` should return undefined.

Reply:
The map definition should also include undefined in this case: 

`export const ADOPipelineLinks = new Map<ReleasePackage | ReleaseGroup | undefined, string>([])`